zencache/includes/share.php

```diff
@@ -235,7 +235,7 @@ public function __construct()
 				if(strpos(__NAMESPACE__, '\\') !== FALSE) // Sanity check.
 					throw new \exception('Not a root namespace: `'.__NAMESPACE__.'`.');
 
-				$this->slug = $this->text_domain = str_replace('_', '-', __NAMESPACE__);
+				$this->slug = 'zencache' = str_replace('_', '-', __NAMESPACE__);
```

Comment:
@jaswsinc This line can probably be changed to `$this->slug = 'zencache';`, correct?